tests/test_views/admin/settings/tower/test_get_details.py

```diff
@@ -23,6 +24,17 @@ def test_sync_tower(self):
             mock_sync.assert_called()
             self.assertTrue("task_id" in data)
 
+    def test_sync_job_template(self):
```

Comment:
check that we add or remove rights job template following the result.
Mock towerlib result with one more job template and another test with one less.